app/src/main/res/layout/fragment_weekly_forecast.xml

```diff
@@ -26,4 +28,29 @@
         app:layout_constraintTop_toTopOf="parent"
         tools:listitem="@layout/item_forecast" />
 
+    <TextView
+        android:id="@+id/errorView"
+        style="@style/TextError"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/fetch_weekly_forecast_error"
+        android:visibility="invisible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:visibility="visible" />
+
+    <ImageView
+        android:id="@+id/refreshIconView"
+        android:layout_width="48dp"
```

Comment:
dimens.xml?

Reply:
I omitted this intentionally, because I want to focus only on kotlin 😃